tests/test_execute/test_basic.xml

```diff
@@ -0,0 +1,5 @@
+<JupyterCellNode classes="jupyter_cell" code_below="False" emphasize_lines="" hide_code="False" hide_output="False" linenos="False" linenostart="True" raises="True" stderr="False">
```

Comment:
here's an example of the file that pytest-regressions uses to decide whether it should fail or not. in subsequent runs, if running the test results in a *different* structure than this XML file, then the test will fail

there is one file created per call to `file_regression.check()`

Reply:
This looks very cool. I wonder though if it would be easier if the XML were included in the test file itself (read as text stream) rather than in separate files. What do you think?